### DISCUSSION TITLE
lwIP or cndp

### DISCUSSION THREAD

**Discussion by lperkov** [unikraft/unikraft#769] [General]
Is the lwIP the only IP stack used within unikraft? Based on the feedback received [here](https://github.com/CloudNativeDataPlane/cndp/issues/275) I'm curious if it would it be possible to add IP stack developed as part of CNDP project?

**Comment by razvand** [comment]
@lperkov, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
